packages/expo-local-authentication/android/src/main/java/expo/modules/localauthentication/LocalAuthenticationModule.java

```diff
@@ -240,6 +244,25 @@ public void run() {
     });
   }
 
+  @Override
+  public void onActivityResult(Activity activity, int requestCode, int resultCode, Intent data) {
```

Comment:
have you tested this on android devices on android 7, 8 and 9.
Promise is not being resolved on such devices when calling .
``` LocalAuthentication.authenticateAsync```
I had to rollback to old version(9.2.0) when things worked fine

Reply:
Same here